internal/buffer/backup.go

```diff
@@ -77,7 +77,7 @@ func (b *Buffer) KeepBackup() bool {
 
 // Backup saves the current buffer to BackupDir()
 func (b *Buffer) Backup() error {
-	if !b.Settings["backup"].(bool) || b.Path == "" || b.Type != BTDefault {
+	if !b.Settings["backup"].(bool) || b.Path == "" || b.Type != BTDefault || !b.requestedBackup {
```

Comment:
I like the idea with the serialization of the save and backup triggers. As of now this should have the least impact on the UI handling and buffer editing.
Let us try this approach.

Reply:
> like we already do for `b.fini` (though that is still ugly, I'd prefer a channel-based solution instead, unless it's too complicated).

`b.fini` could be removed with a further "fini" channel, but I don't think that the code becomes more readable or less complex by this, since `backupRequestChan` must be cleared and filled again with the non-fini requests left.